### PR TITLE
Update obsolete error

### PR DESCRIPTION
**Pull request by HaoK**
Fixes https://github.com/aspnet/Security/issues/1334

@Tratcher @moozzyk

### CODE REVIEW

File: src/Microsoft.AspNetCore.Authentication.Google/GoogleAppBuilderExtensions.cs [hunk 1]
```diff
@@ -14,7 +12,7 @@ namespace Microsoft.AspNetCore.Builder
     public static class GoogleAppBuilderExtensions
     {
         /// <summary>
-        /// Obsolete, see https://go.microsoft.com/fwlink/?linkid=845470
+        /// UseGoogleAuthentication is obsolete. Configure Google authentication with AddAuthentication().AddGoogle in ConfigureServices. See https://go.microsoft.com/fwlink/?linkid=845470 for more details.
         /// </summary>
         /// <param name="app">The <see cref="IApplicationBuilder"/> to add the handler to.</param>
         /// <returns>A reference to this instance after the operation has completed.</returns>
```
Comment (Tratcher) [inline src/Microsoft.AspNetCore.Authentication.Google/GoogleAppBuilderExtensions.cs]: Missing the Obsolete attribute for this one?

File: src/Microsoft.AspNetCore.Authentication.Facebook/FacebookAppBuilderExtensions.cs [hunk 2]
```diff
@@ -23,7 +23,7 @@ public static IApplicationBuilder UseFacebookAuthentication(this IApplicationBui
         }
 
         /// <summary>
-        /// Obsolete, see https://go.microsoft.com/fwlink/?linkid=845470
+        /// UseFacebookAuthentication is obsolete. Configure Facebook authentication with AddAuthentication().AddFacebook in ConfigureServices. See https://go.microsoft.com/fwlink/?linkid=845470 for more details.
```
Comment (Tratcher) [inline src/Microsoft.AspNetCore.Authentication.Facebook/FacebookAppBuilderExtensions.cs]: Should this also go in the attribute?

File: src/Microsoft.AspNetCore.Authentication.Google/GoogleAppBuilderExtensions.cs (hunk 1 above)
Comment (HaoK) [inline src/Microsoft.AspNetCore.Authentication.Google/GoogleAppBuilderExtensions.cs]: Looks like it, added

File: src/Microsoft.AspNetCore.Authentication.Facebook/FacebookAppBuilderExtensions.cs (hunk 2 above)
Comment (HaoK) [inline src/Microsoft.AspNetCore.Authentication.Facebook/FacebookAppBuilderExtensions.cs]: Updated all the attribute errors now

**Pull request merged**